app/src/main/java/com/example/makeiteven2/fragments/FragmentStartScreen.kt

```diff
@@ -14,16 +14,18 @@ import android.widget.ImageButton
 import android.widget.ImageView
 import androidx.fragment.app.Fragment
 import com.example.makeiteven2.R
+import com.example.makeiteven2.extras.OnTouchAnimation
 import kotlinx.android.synthetic.main.fragment_start_screen.view.*
 
 class FragmentStartScreen : Fragment() {
 
     private lateinit var callBack : OnButtonClicked
     private lateinit var stageModeBtn : Button
-    lateinit var arcadeModeBtn : Button
-    lateinit var tutorialBtn : Button
-    lateinit var scoreBoardBtn : ImageButton
-    lateinit var logoIv : ImageView
+    private lateinit var onTouchAnimation : OnTouchAnimation
```

Comment:
private lateinit var onTouchAnimation : OnTouchAnimation
    private lateinit var arcadeModeBtn : Button
    private lateinit var tutorialBtn : Button
    private lateinit var scoreBoardBtn : ImageButton
    private lateinit var logoIv : ImageView

and m before members name

Reply:
done